cse-cc-import-runner-app/src/main/resources/application.yml

```diff
@@ -10,30 +10,31 @@ spring:
       reply-timeout: 2400000
   cloud:
     function:
-      definition: interrupt;request
+      definition: softInterrupt;request
     stream:
       default:
         binder: rabbit
       bindings:
-        interrupt-in-0:
-          destination: cse-interrupt
+        softInterrupt-in-0:
+          destination: cse-x-stop-task # replace 'x' with the process: import-d2cc, import-idcc...
         request-in-0:
-          destination: cse-import-idcc-request
+          destination: cse-x-request # replace 'x' with the process: import-d2cc, import-idcc...
+          group: cse-x-runner # replace 'x' with the process: import-d2cc, import-idcc...
         request-out-0:
-          destination: cse-response
+          destination: cse-x-response # replace 'x' with the process: import-d2cc, import-idcc...
         task-status-update:
-          destination: cse-import-idcc-task-status-updated
+          destination: cse-x-task-status-updated # replace 'x' with the process: import-d2cc, import-idcc...
```

Comment:
Since process value (eg. d2cc) is repeated several times, it might be better to define a property with this value and then reference it. For instance : process: import-d2cc and then destination: cse-${process}-stop-task